src/poetry/core/constraints/version/version_range.py

```diff
@@ -58,13 +58,25 @@ def is_any(self) -> bool:
     def is_simple(self) -> bool:
         return self._min is None or self._max is None
 
+    @property
+    def is_inclusive(self) -> bool:
+        return self._include_min or self.include_max
```

Comment:
Isn't the naming a little ambiguous? Why `or` and not `and`? Further, the `or self.include_max` part seems not to be required by any test.

Reply:
Suggestions welcome. 

I just followed the names inclusive/exclusive ordering from pep440 really. But I guess in the context of ranges it can be ambiguous.

I could just drop the property altogether.

Reply:
I would just drop it. (The naming is clear as long as there is only a lower limit or only an upper limit, but not if there are both.)